lib/async.js

```diff
@@ -46,7 +46,7 @@ module.exports = function resolve (x, opts, cb) {
     else loadNodeModules(x, y, function (err, n, pkg) {
         if (err) cb(err)
         else if (n) cb(null, n, pkg)
-        else if (core[x]) return cb(null, x);
+        else if (core[x]) return cb(null, x, null);
```

Comment:
Hmm - I'm not super comfortable about working around what is definitely a bug - for convenience or no - in `co`.

This also is a breaking change for anyone who's currently testing `=== undefined` or similar.